models/postexport/columns.yaml

```diff
@@ -13,6 +13,7 @@ columns:
     categories: rainlab.blog::lang.post.categories
     author_email: rainlab.blog::lang.post.author_email
     featured_image_urls: rainlab.blog::lang.post.featured_images
+    published: rainlab.blog::lang.post.published
```

Comment:
@mjauvin Minor point, but could this be moved to above the `published_at` line, to group the fields together?

Reply:
done